core/src/main/scala/org/apache/spark/storage/BlockManagerMessages.scala

```diff
@@ -32,8 +32,13 @@ private[spark] object BlockManagerMessages {
   // blocks that the master knows about.
   case class RemoveBlock(blockId: BlockId) extends ToBlockManagerSlave
 
+  // Replicate block excluding certain executors for graceful shutdown.
```

Comment:
Adding this comment makes the whole comment block a little bit confusing. (As in, which one is correct?)

If you want to document the `excluding` parameter you can create a scaladoc comment instead. Mentioning "graceful shutdown" here doesn't really add a lot of information.

Reply:
Removed both comments because the old one isn't accurate anymore.